go.mod

```diff
@@ -197,6 +196,7 @@ require (
 	go.uber.org/atomic v1.10.0 // indirect
 	go.uber.org/dig v1.15.0 // indirect
 	go.uber.org/fx v1.18.2 // indirect
+	go.uber.org/multierr v1.9.0 // indirect
```

Comment:
Can we remove this dependency altogether? (go.sum) should be updated too

Reply:
I assume another import from uber uses the multierr as a dep so we cannot get rid of it altogether until they update

Reply:
@dylanlott Can you verify?

Reply:
It's being imported by some other dependency of ours, but there are no more usages of multierr in our specific code that I could find.

Reply:
> Can we remove this dependency altogether? (go.sum) should be updated too

I actually don't think go.sum needs to change, the dependency was only made indirect, and the version stayed the same, so I suspect the dependency graph stayed the same 🤔